README.md

```diff
@@ -29,18 +29,144 @@ graph TD
     Platform --> Platform-Prod
 ```
 
-## Creating a new subscription
+## New subscription
+
+### Naming the subscription
+
+First you will need to decide on a name for the new subscription you want.
+
+The format is `${department}-${service}-${environment}`
+
+Department is normally `DTS`
+For all of HMCTS e.g. CFT, Crime, SDS, then use `HMCTS`.
+
+Service is a short name that normally maps to the project or group of projects.
+e.g. BAR (banking and returns) or CFT, (Civil, Family and Tribunals group of projects).
+
+Environment is one of the allowed environments names:
+- SBOX (Sandbox)
+- TEST
```

Comment:
I am assuming we are looking to remove INTSVC naming ?

Reply:
yes it's deprecated and should not be used for new ones, intentionally left out, when I implement naming standards automation existing subscriptions can be excluded from that